sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -140,6 +150,16 @@ def test_Beam():
     q = x*(E*I*f - w0*SingularityFunction(e, a1, 4)/24 - w2*SingularityFunction(e, c1, 2)/2)/(E*I) + (w0*SingularityFunction(x, a1, 5)/120 + w2*SingularityFunction(x, c1, 3)/6)/(E*I) + (E*I*(-c*f + d) + c*w0*SingularityFunction(e, a1, 4)/24 + c*w2*SingularityFunction(e, c1, 2)/2 - w0*SingularityFunction(c, a1, 5)/120 - w2*SingularityFunction(c, c1, 3)/6)/(E*I)
     assert p == q
 
+    # Test for shear stress distribution function
+    p = b2.shear_stress()
+    q = w0*SingularityFunction(x, a1, 5)/2 + w2*SingularityFunction(x, c1, 4)
+    assert p == q
+
+    # Test for axial stress distribution function
+    p = b2.axial_stress()
```

Comment:
I think you have added this method inside `Beam3D` class, why you using it over `Beam`?